tests/integration/test_machine.py

```diff
@@ -1,5 +1,6 @@
 import asyncio
 import pytest
```

Comment:
pytest shoudl go in the second import group.

Reply:
done